src/stdlib/io.rs

```diff
@@ -9,19 +9,34 @@ use std::{io::Write, ops::Deref};
 fndr_native_func!(
     /// Prints to stdout
     print_func,
-    |_, item| {
+    |_, item, argv| {
         print!("{}", item.to_string());
-        let mut lock = std::io::stdout().lock();
-        let _ = lock.flush();
+        let argv = match &*argv {
+            List(l) => l.deref(),
+            _ => unreachable!(),
```

Comment:
Can we add a message to the unreachable statement? Like `unreachable!("Last argument is always a vararg list")`